snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for daily stats ping, tracking instance metrics and settings",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "instance_stats",
+    "format": "jsonschema",
+    "version": "1-0-0"
```

Comment:
version here is mismtached, should be `1-0-1` right?

Reply:
fixed.